docs/markdown/glossary.md

```diff
@@ -0,0 +1,260 @@
+
+## Hop 
+
+The movement of an ILP packet from one router to another as it travels through the Interledger network to reach its destination.                                                                         
+
+## IL-RFC  
+
+Interledger request for comments. A document describing a part of the protocol stack, or a topic related to it, such as this glossary. The official list of IL-RFCs is maintained in gh:interledger/rfcs.                                                                                                                               
+
+## Interledger (in "Let's use Interledger for that!")  
+
+The Interledger protocol stack.                                                                         
+
+## interledger (in "An interledger network")
+
+A network of two or more ledgers that use the Interledger protocol stack. 
+
+See also: [Interledger (The)](#interledger-in-the-interledger) 
+
+The adjective 'interledger' is also sometimes used to refer to transactions that cross multiple ledgers, even if ILP is not used (similar to 'international' meaning 'across multiple nations').                                                                         
+
+## Interledger (in "The Interledger")  
+
+The public network of ledgers connected via the Interledger protocol stack.                                                                         
+
+## Interledger Address 
+
+An Interledger address provides a way to route ILP packets to their intended destination through a series of hops, including any number of ILP routers. 
+
+An Interledger address is a dot-separated string that contains prefixes to group ledgers. An example address might look like `g.us.acmebank.acmecorp.sales.199` or `g.crypto.bitcoin.1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2`. 
+
+
+Note that anyone can claim to have a certain Interledger address, there is no registry of them. Whether or not a payment ends up at the intended receiver is ultimately safeguarded by the hashlock condition, not by enforcement of address ownership. 
+
+See also: [IL-RFC 15](https://github.com/interledger/rfcs/blob/master/0015-ilp-addresses/0015-ilp-addresses.md)                                                                      
+
+## Interledger Architecture    
+
+The Interledger protocol stack and the design principles behind it.
+
+See also: [IL-RFC 1](https://github.com/interledger/rfcs/blob/master/0001-interledger-architecture/0001-interledger-architecture.md)                                                                       
+
+## Interledger Layer   
+
+The second layer of the Interledger protocol stack, consisting of the Interledger Protocol version 4 (ILPv4). This layer is concerned with currency amounts, routing, and whether each step in a payment arrives in time or expires. This protocol finds a path to connect a sender and receiver using any number of intermediaries.                                                                     
```

Comment:
Are we still counting the ledger as layer 1? If so ILP would be layer 3

Reply:
Looking at this diagram, I am assuming that the grey portion comprises the Interledger layers. That's why I have mentioned ILP as the second layer. Is that diagram outdated?

Reply:
I think what @sharafian is referencing is the protocol layering outlined in [this blog post](https://medium.com/xpring/layer-3-is-for-interoperability-ca387fa5f7e2) by @emschwartz. 

In that post, Interledger protocols are layer-3 and "up" (i.e., layers 4 and 5 too). I think we should maintain _that_ layering because it accounts for related technologies that aren't Interledger per-se, like Bitcoin, which would be a Layer-1 ledger.

Reply:
I have removed the numbering from this definition to avoid confusion.
https://github.com/interledger/interledger.github.io/pull/123/files#diff-0f4b6322d1459dbe5c7f5eb93d2cd2efR117